easycord/composer.py

```
@@ -128,11 +128,21 @@ def add_plugin(self, plugin: Plugin) -> Composer:
 
+    def add_plugins(self, *plugins: Plugin) -> Composer:
+        """Queue several plugins to be added to the bot."""
+        self._plugins.extend(plugins)
+        return self
+
     def add_group(self, group) -> Composer:
         """Queue a SlashGroup to be added to the bot."""
         self._groups.append(group)
         return self
 
+    def add_groups(self, *groups) -> Composer:
```

Ruff (0.15.10)

[warning] Missing type annotation for `*groups`

(ANN002)